app/views/hyrax/autopopulation/dashboard/work_fetchers/_hyku_autopopulation_settings_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for(current_account, url: hyrax_autopopulation.settings_work_fetchers_path(current_account), :html => { class: 'form control' }) do |f| %>
+    <div class="panel-body">
+     <div class="form-group">
+       <%=f.fields_for :settings do |ff| %>
+         <%= ff.label :orcid_ids_for_autopopulation %><br>
+         <%= ff.text_field :orcid_list, :value => account.settings.dig("autopopulation", "orcid_list"), class: 'form-control', id: "doi_autopopulation_settings" %><br>
```

Comment:
These inputs (and the other form partial) could use `value: ...`

Reply:
Ok noted